cmd/virt-launcher/virt-launcher.go

```diff
@@ -63,12 +63,11 @@ func init() {
 	libvirt.EventRegisterDefaultImpl()
 }
 
-func markReady(readinessFile string) {
-	f, err := os.OpenFile(readinessFile, os.O_RDONLY|os.O_CREATE, 0666)
+func markReady() {
```

Comment:
now what happens if the compute container manages to hit this point before the containerDisks create their socket? Will the reconcile loop handle that as a sync error? maybe we could wait for all container disk sockets to appear before compute is ready

Reply:
Done. virt-handler now retries a few times without throwing errors before retrying noisy and with a backoff.